internal/policies/mount/entries_test.go

```diff
@@ -0,0 +1,58 @@
+package mount
+
+import (
+	"fmt"
+
+	"github.com/ubuntu/adsys/internal/policies/entry"
+)
+
+// EntriesForTests is a map with multiple []entries combinations to be used in tests.
+var EntriesForTests = map[string][]entry.Entry{
```

Comment:
I would add one entry with a tag (anonymous) btw (could be a separate test case too for both unit and package test). That way, we show we don’t do anything with them and it’s a common case we will get.